examples/cifar10/pt/networks/cifar10_nets.py

```diff
@@ -85,6 +85,7 @@ def __init__(self):
             nn.Conv2d(in_channels=256, out_channels=256, kernel_size=3, padding=1),
             nn.ReLU(inplace=True),
             nn.MaxPool2d(kernel_size=2, stride=2),
+            nn.Flatten(),  # added to support split learning
```

Comment:
would this layer added affects other cifar10 examples?

Reply:
Ok. I'll try to add this to the split learning logic rather than to the network itself.